### PR TITLE
feat(multicontent): Saved search can be added to multicontent view

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/app/scripts/superdesk-desks/aggregate.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+                   this.searches = searchesList._items;
+                   _.each(this.searches, function(item) {
+                       self.searchLookup[item._id] = item;
+                   });
+               }));
+        }))
+        .then(angular.bind(this, function() {
+            return preferencesService.get(PREFERENCES_KEY)
+                .then(angular.bind(this, function(prefference) {
+                    this.groups = prefference != null && prefference.groups ? prefference.groups : [];
```
Comment (amagdas) [inline client/app/scripts/superdesk-desks/aggregate.js]: Small typo here, `prefference`.

File: client/app/scripts/superdesk-desks/views/item-preview.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div class="preview-container" ng-init="contenttab = true">
+    <header>
+        <ul class="nav nav-tabs">
+            <li ng-class="{active: contenttab}">
+                <button ng-click="contenttab = true">Content</button>
```
Comment (amagdas) [inline client/app/scripts/superdesk-desks/views/item-preview.html]: Translate missing.

File: client/app/scripts/superdesk-desks/views/item-preview.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<div class="preview-container" ng-init="contenttab = true">
+    <header>
+        <ul class="nav nav-tabs">
+            <li ng-class="{active: contenttab}">
+                <button ng-click="contenttab = true">Content</button>
+            </li>
+            <li ng-class="{active: !contenttab}">
+                <button ng-click="contenttab = false">Metadata</button>
```
Comment (amagdas) [inline client/app/scripts/superdesk-desks/views/item-preview.html]: Translate

**Comment by amagdas** [comment]
:+1:  aside from the small comments

**Comment by plamut** [comment]
:+1: 

Some `JsDoc` comments would be welcome, but otherwise it seems good in general.

**Pull request merged**